api/models/app.go

```diff
@@ -403,6 +405,14 @@ func (a *App) ExecAttached(pid, command string, rw io.ReadWriter) error {
 	go io.Copy(iw, rw)
 	go io.Copy(rw, or)
 
+	success := make(chan struct{})
+
+	go func() {
+		<-success
+		d.ResizeExecTTY(id, height, width)
```

Comment:
Will this TTY setting persist beyond the life of this goroutine? If so any side effects to consider?